src/gwt/src/org/rstudio/studio/client/common/dependencies/DependencyManager.java

```diff
@@ -966,6 +967,21 @@ private void installDependencies(final String context,
                                     final boolean silentEmbeddedUpdate,
                                     final CommandWithArg<Boolean> onComplete)
    {
+    
+      // Command to run when dependency installation is complete
+      CommandWithArg<Boolean> onCompletion = satisfied ->
+      {
+         // Confirmed: dependencies are in place. Switch
+         // away from the Jobs tab to put the user back in
+         // context.
+         if (satisfied)
+         {
+            events_.fireEvent(new ConsoleActivateEvent(false));
```

Comment:
IMHO we should only do this if the Jobs tab is open and currently focused; otherwise we might steal focus from a user who's now working in the IDE on other stuff while installation goes on.

Reply:
That's a real risk, but unfortunately it's not so easy to detect whether the user has disturbed the state. We don't send focus to the jobs pane when the job starts running. Some refinement would be nice but is out of scope for 1.3.